docs/conf.py

```diff
@@ -16,13 +16,13 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#
 import os
 import sys
-sys.path.insert(0, os.path.abspath('../'))
+from recommonmark.parser import CommonMarkParser
```

Comment:
Missing packages needed for docs building: `recommonmark`, `sphinx_autodoc_typehints`, `sphinx_rtd_theme`, `sphinx_click`
They should be added in `requirements-dev.txt`